.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": [
+    "plugin:bpmn-io/es6",
```

Comment:
Suggestion: Let's update to the latest version of `eslint-plugin-bpmn-io` and use `plugin:bpmn-io/browser` config instead. Then, you can remove the `env` section as well.